mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -166,6 +169,38 @@ def __init__(self, name=None, bus=None, use_settings=True):
         # Skill Public API
         self.public_api = {}
 
+        if bus:
+            self._startup(bus)
+
+    def _startup(self, bus, skill_id=""):
```

Comment:
Should this be an internal method? The only call is from an external module

Reply:
it is an internal method in parent PR, the reasoning is that if core adopts this it simplifies skill loading, however as commented below no current skill actually allows passing the bus in init method... so this is currently always called externally until mycroft-core makes up their mind. 

The reason for it to be private is that no skill developer ever should touch this method, this indicates that to developers